doc/testing.md

```diff
@@ -1,30 +0,0 @@
-# Testing
-
-(This page is about testing Chestnut itself, *not* about how to do testing in your Chestnut app.)
```

Comment:
Renaming it might make more sense or putting it in a "chestnut development" section or something